libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -286,6 +286,35 @@ void AP_Camera_Backend::send_camera_fov_status(mavlink_channel_t chan) const
 }
 #endif
 
+// send camera capture status message to GCS
+void AP_Camera_Backend::send_camera_capture_status(mavlink_channel_t chan) const
+{
+    const float NaN = nanf("0x4152");
+
+    const uint32_t time_boot_ms = AP_HAL::millis();
+    const uint8_t video_status = 0;       // Current status of video capturing (0: idle, 1: capture in progress)
+    const uint32_t recording_time_ms = 0; // Time since recording started (ms)
+    const float available_capacity = NaN; // Available storage capacity (MiB)
+
+    // Current status of image capturing (0: idle, 1: capture in progress, 2: interval set but idle, 3: interval set and capture in progress)
+    const uint8_t image_status = (time_interval_settings.num_remaining > 0) ? 2 : 0;
+    // Image capture interval (s)
+    const float image_interval = static_cast<float>(time_interval_settings.time_interval_ms) / 1000.0;
+    // Total number of images captured
+    const int32_t image_count = this->image_index;
```

Comment:
It's just a style thing but maybe we can get rid of some of these intermediate variables and just directly place them in the "camera_capture_status_send(" section below?  Then put the comments beside each field.  "image_status" is the only one that is complex enough to be worth breaking out I think.

    mavlink_msg_camera_capture_status_send(
        chan,
        AP_HAL::millis(),
        image_status,
        0,                                // current state of video capturing (0: idle, 1: capture in progress)
        time_interval_settings.time_interval_ms / 1000.0,  // Image capture interval (s)
        0,                                // time since recording started (ms)
        NaN,                           // available storage capacity (MiB), NaN if unknown
        image_index);             // total number of images captured

Reply:
Updated.